R/module_teal.R

```diff
@@ -201,7 +201,7 @@ srv_teal <- function(id, data, modules, filter = teal_slices()) {
 
     data_validated <- .trigger_on_success(data_pulled)
 
-    data_rv <- reactive({
+    data_validated_r <- reactive({
```

Comment:
Suddenly `_r` appears. I propose:

`data_pulled` -> `data_handled` (to follow object name returned from srv_init_data)
`data_validated` -> `data_triggered` (or keep `data_validated`)
`data_validated_r` -> `data_signatured`